samples/eth_inrapD.json

```diff
@@ -0,0 +1,16 @@
+{
+  "_id": "6b56ee657c870dfacd34e9ae4e0643dd",
+  "links": [
+    {
+      "verb": "isGroundedIn",
+      "object": "6b56ee657c870dfacd34e9ae4e0576ce"
+    }
+  ],
+  "dc_license": "https://creativecommons.org/licenses/by/4.0/",
+  "text": "Lors d'une de mes premières venues à l'atelier, le nettoyage de la carrosserie d'une locomotive à vapeur m'est confié. Pendant ce temps, Luc, un autre bénévole la met en chauffe. Il s'agit d'allumer le feu dans le foyer de manière à progressivement réchauffer l'eau présente dans la chaudière, qui, se transformant en vapeur, fournit la force motrice d'une telle locomotive. \n \nAssez rapidement, plusieurs personnes se présentent à moi. Ayant identifié que je suis nouveau, elles me souhaitent la bienvenue. Un premier membre me trouve un seau plus grand que celui avec lequel j'ai commencé le nettoyage. Un autre m'indique que j'aurais intérêt à commencer par les parties hautes de la locomotive, plutôt que des côtés, car celles-ci deviendront vite très chaudes (je risque de me bruler si j'attends pour les laver). \n \nLa chauffe dure à trois à quatre heures. Une fois que la locomotive est prête, des essais ont lieu aux abords de l'atelier. Pour réaliser ces essais, Leo se joint à Luc. Ces essais consistent à faire quelques manœuvres au sein du faisceau (rail et aiguillage) aux alentours de l'atelier. Très vite, Leo descend de la locomotive, plonge son bras sous la carrosserie, et affichant une moue crispée, affirme que les boîtes d'essieu chauffent trop. \n \nIl est midi. Nous interrompons tous les trois les essais et rejoignons les autres bénévoles pour le repas de midi. La météo étant clémente, nous sortons quelques chaises pour manger en plein air. Tout en mangeant, Luc et Leo exposent les résultats des essais. Les différentes personnes présentes discutent des solutions possibles. J'apprends alors que le problème est connu et récurrent sur cette ancienne locomotive de charbonnage. Afin de faire diminuer ces *hot boxes*, les bénévoles ont déjà testé différents types d'huiles de plus en plus épaisses. C'est un des administrateurs de l'association, travaillant dans l'industrie, qui a identifié l'huile épaisse utilisée lors des essais. Un autre convive m'explique que cette huile est tellement épaisse qu'elle donne une sensation de velour lorsqu'on la fait glisser entre les doigts.",
+  "dc_issued": "2012",
+  "dc_language": "fra",
+  "dc_title": "Treignes, le 8 septembre 2012",
+  "dc_creator": "Christophe Lejeune",
+  "couchapp": {}

```

Comment:
This is an artefact from pushing the document with the command.  This shouldn't be in the original sample.